File: Hivemind/Contracts/GangerLevelUpReport.cs
using Hivemind.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hivemind.Contracts
{
    public class GangerLevelUpReport
    {
        public string GangerName { get; set; }
        public string Description { get; set; }
        public IEnumerable<SkillType> NewSkillFromCategory { get; set; }
    }
}

File: Hivemind.Ng/src/autogenerated/entities/GangerLevelUpReport.ts

export class GangerLevelUpReport {
    public gangerName: string;
    public description: string;
    public newSkillFromCategory: any;

    public constructor(partial: Partial<GangerLevelUpReport>) {
        if (partial.gangerName) {
            this.gangerName = partial.gangerName;
        }
        if (partial.description) {
            this.description = partial.description;
        }
        if (partial.newSkillFromCategory) {
            this.newSkillFromCategory = partial.newSkillFromCategory;
        }
    }
}